### ISSUE TITLE
ipywidgets in jupyterlab: Missing widget controls and "Error displaying widget"

### ISSUE THREAD

**Issue by CursingRabbit** [jupyterlab/jupyterlab#4127]
ipywidgets are not working in jupyterlab, which are working in jupyter notebook. I am using the latest version of jupyterlab and extensions:

```
jupyter labextension list
JupyterLab v0.31.11
Known labextensions:
   app dir: /home/user/PycharmProjects/coherence/venv/share/jupyter/lab
@jupyter-widgets/jupyterlab-manager
        @jupyter-widgets/jupyterlab-manager v0.33.2  enabled  OK
jupyter-matplotlib
        jupyter-matplotlib v0.3.0  enabled  OK
```

```
import matplotlib as mpl
import matplotlib.pyplot as plt

import ipywidgets
from ipywidgets import interact, interactive, fixed, interact_manual, Button, VBox, HBox, interactive

import numpy as np
```

A slider widget will show up:

![screen shot 2018-03-06 at 17 58 52](https://user-images.githubusercontent.com/15199213/37049566-20f05034-2169-11e8-9600-8c76fe716cae.png)

But using interact will not create a checkbox in the following case:
![screen shot 2018-03-06 at 17 59 28](https://user-images.githubusercontent.com/15199213/37049616-4f99eab2-2169-11e8-8165-f88f59ecdc78.png)

Using interact and matplotlib creates also no controls:
![screen shot 2018-03-06 at 18 06 13](https://user-images.githubusercontent.com/15199213/37049674-8c04d5b6-2169-11e8-827d-7fed4e2cc633.png)

Using interactive, there is an error message "Error displaying widget" which disappears after a short time, followed only by the plot without any controls:
![screen shot 2018-03-06 at 18 33 59](https://user-images.githubusercontent.com/15199213/37050950-23966518-216d-11e8-8dc7-224a3af6f93f.png)

Error messages in the web console of the browser:
```
Error displaying widget  main.ed321867e6f4b12eff75.js:1:815713
Module @jupyter-widgets/output, semver range 1.1.0 is not registered as a widget module  main.ed321867e6f4b12eff75.js:1:815752
Error: Object 'jupyter.widget' not found in registry main.ed321867e6f4b12eff75.js:1:1288299
uncaught exception: Module @jupyter-widgets/output, semver range 1.1.0 is not registered as a widget module  (unknown)
Error: Object 'jupyter.widget' not found in registry main.ed321867e6f4b12eff75.js:1:1288299
uncaught exception: Module @jupyter-widgets/output, semver range 1.1.0 is not registered as a widget module (unknown)
Error: Object 'jupyter.widget' not found in registry main.ed321867e6f4b12eff75.js:1:1288299
uncaught exception: Module @jupyter-widgets/output, semver range 1.1.0 is not registered as a widget module  (unknown)
Error: Object 'jupyter.widget' not found in registry main.ed321867e6f4b12eff75.js:1:1288299
```
In issue #3263 the fix was to update to the latest versions, but this is not helping here.

**Comment by jasongrout** [comment]
I just tried a fresh install with those versions of jupyterlab and `@jupyter-widgets/jupyterlab-manager` and your first interact example worked. Perhaps there was a problem in upgrading. That error you're seeing usually happens if the jupyterlab widgets extension is too old.

1. What version of ipywidgets do you have installed in the kernel?
2. Can you uninstall the `@jupyter-widgets/jupyterlab-manager` extension and reinstall it?

**Comment by ktuluv** [comment]
Hello Jason, 

I am facing the same issue. 
Here is the output when run  >jupyter labextension list

JupyterLab v0.31.12
Known labextensions:
   app dir: C:\Users\vargas.s.12\AppData\Local\Continuum\anaconda3\envs\py36\share\jupyter\lab
@jupyter-widgets/jupyterlab-manager
        @jupyter-widgets/jupyterlab-manager v0.33.2 enabled  ok

**Comment by Geekly** [comment]
I'm having the same issue. Posting it here in case you want to track compatible versions.
```$ jupyter labextension list
JupyterLab v0.35.4
Known labextensions:
   app dir: c:\miniconda\envs\py36\share\jupyter\lab
        @jupyter-widgets/jupyterlab-manager v0.38.1 enabled  ok
        jupyter-matplotlib v0.3.0 enabled  ok
        qgrid v1.1.1 enabled  ok
```

**Comment by jasongrout** [comment]
@Geekly - that looks like correct versions to work. Do you see any javascript errors in the browser javascript console?

**Comment by caozhen-alex** [comment]
@sebvargo @Geekly 
Did you solve this porblem?
I face the same problem too.

**Comment by jasongrout** [comment]
If you can upgrade, can you try jlab 1.0 and the newest version of `@jupyter-widgets/jupyterlab-manager` jlab extension with ipywidgets 7.5?

**Comment by pubmania** [comment]
I can confirm the behavious. Works on Jupyter Notebook but not on Jupyter Lab.

**Comment by grisaitis** [comment]
This might be fixed in version >= 1.1, out last week. 

Related: https://github.com/jupyterlab/jupyterlab/issues/4855#issuecomment-524324129
